scripts/module-tests.sh

```diff
@@ -21,7 +21,7 @@ execute_mod_tests() {
 
     echo ">>> running $go_mod tests"
     cd $mod_dir;
-    go test -mod=readonly -timeout 30m -coverprofile=${root_dir}/${coverage_file}.tmp -covermode=atomic -tags='norace ledger test_ledger_mock rocksdb_build'  ./...
+    GOWORK=off go test -mod=readonly -timeout 30m -coverprofile=${root_dir}/${coverage_file}.tmp -covermode=atomic -tags='norace ledger test_ledger_mock rocksdb_build'  ./...
```

Comment:
This is what give this PR it's title.

Reply:
Which PR checks call into this script? I vote yes for all CI.

Reply:
test-submodules